service-providers/network-requester/src/config/mod.rs

```diff
@@ -81,8 +81,13 @@ impl NymConfigTemplate for Config {
 
 impl Config {
     pub fn new<S: AsRef<str>>(id: S) -> Self {
+        let mut base = BaseClientConfig::new(id.as_ref(), env!("CARGO_PKG_VERSION"));
+        // By default we have no cover traffic, but we have a slow trickle of cover traffic
+        // packages acting as keepalive messages.
+        base.set_no_cover_traffic_with_keepalive();
```

Comment:
couldn't we just set all of those as you know, the defaults?

Reply:
But we don't want them to be default for BaseClientConfig, only for when BaseClientConfig is used in the NR